pos_print_receipt_backend/views/pos_order_view.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_pos_order_form" model="ir.ui.view">
+        <field name="model">pos.order</field>
+        <field name="inherit_id" ref="point_of_sale.view_pos_pos_form"/>
+        <field name="arch" type="xml">
+            <field name="state" position="before">
+                <button name="%(action_report_receipt)d" string="Receipt" attrs="{'invisible': [('state', '=', 'draft')]}" type="action" groups="point_of_sale.group_pos_user"/>
```

Comment:
a button and an email template to send it by email would be great ! it make it paperless and especially if the customer request the ticket by email and not in the shop.